src/pages/eunjungPark/Login/Login.js

```diff
@@ -1,8 +1,114 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Login.scss';
 
 class LoginEunjung extends Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  handleInput = event => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+  };
```

Comment:
- 하나의 함수로 두 가지 인풋을 처리해주신 것 너무 좋네요 ✨
- 구조분해할당도 해주시면 더 좋을 것 같습니다.
```suggestion
  handleInput = event => {
    const { name, value } = event.target;
    this.setState({
      [name]: value,
    });
  };
```